aospy/calc.py

```diff
@@ -897,7 +897,7 @@ def _load_from_scratch(self, dtype_out_time, dtype_out_vert=False,
 
     def _load_from_archive(self, dtype_out_time, dtype_out_vert=False):
```

Comment:
Let's rename `_load_from_tar`.  Is any of this logic unique to /archive other than the `dmget` call?  For the time being, add a try/except guard around the dmget call here as well, in case anybody tries using this from outside GFDL.